Add unit tests for LongHashSet, ListUtils and CharsetUtils

Braavos6 opened the pull request:
I've analysed your codebase and noticed that `qunar.tc.qmq.delay.base.LongHashSet` , `qunar.tc.qmq.utils.ListUtils` and `qunar.tc.qmq.utils.CharsetUtils` is not fully tested.
I've written some tests for the methods in this class with the help of [Diffblue Cover](https://www.diffblue.com/opensource).

Hopefully, these tests will help you detect any regressions caused by future code changes. If you would find it useful to have additional tests written for this repository, I would be more than happy to look at other classes that you consider important in a subsequent PR.

petersear reviewed (approved):
You could consider shortening the commit line by listing the unqualified class names and adding the fully-qualified names on a second line. I've seen suggestions that the PR text line (taken from the first commit line) shouldn't be over 80 (or even 50) characters and shouldn't wrap. It seems to be subjective though.

Braavos6 committed:
Add unit tests for LongHashSet, ListUtils and CharsetUtils

Fully-qualified classnames
qunar.tc.qmq.delay.base.LongHashSet
qunar.tc.qmq.utils
qunar.tc.qmq.CharsetUtils

These tests were written using Diffblue Cover.
